main.js

```diff
@@ -0,0 +1,51 @@
+import './style/style.sass'
+import './style/reset.css'
+import { Todo } from './Todo.js'
+import $ from 'jquery'
+
+const inputText = $('.input-text')
+const buttonAddTodo = $('.add-item')
+const filtrAll = $('.filtr-all')
+const filtrTodo = $('.filtr-todo')
+const filtrDone = $('.filtr-done')
+
+buttonAddTodo.click(() => {
+  const todo = new Todo(inputText.val())
+  todo.createNewTodo(inputText.val())
+console.log(todo)
+  inputText.val(null)
+})
+
+export const deleteTodo = (deleteButton) => {
+  deleteButton.click(event => {
+    $(event.target)
+      .closest('li')
+      .remove()
+  })
+}
+
+export const acceptedTodo = (acceptedButton) => {
+  acceptedButton.click(event => {
+    $(event.target)
+      .closest('li')
+      .addClass('checked')
+
+    Todo.isDone = true
```

Comment:
you can't do it. You should access instace of the class like `todo`.
Additionaly you should create a method like `toggleTodo`